tests/functional/test_credentials.py

```diff
@@ -68,13 +69,13 @@ def assert_consistent_credentials_seen(self, creds, func):
             # from the first refresh ('1'), the secret key from
             # the second refresh ('2'), and the token from the
             # first refresh ('1').
-            self.assertTrue(creds[0] == creds[1] == creds[2], creds)
+            assert creds[0] == creds[1] == creds[2]
```

Comment:
Is there a particular you ran into that made you remove the failure messages when porting to `assert`?